README.md

```diff
@@ -40,4 +40,26 @@ open `{universal}/tests/web/html/all-tests.html` in your preferred browser:
 
 Running node-based tests:
 
-    node tests/all-tests.js
\ No newline at end of file
+    node tests/all-tests.js
+
+Docker
```

Comment:
Right now we have _usage_, _installation_ and _testing_ as sections. Do you mind to rename this new one into something like "Deploying with Docker", "Use Docker to deploy ..." or anything that you feel suitable for it?